composer.json

```diff
@@ -14,13 +14,18 @@
         }
     ],
     "bin": [
+        "bin/serviceplatformen-sf1500-data-lookup",
+        "bin/serviceplatformen-sf1514-saml-token",
```

Comment:
Why do we need a command for a SAML token?

Reply:
It was added as a first step towards the sf1500 command and helps confirming whether a SAML-token can be fetched or not. 

I suppose the sf1500 command does this as well, but kept it in case it would be useful later for separately testing SAML-token fetching.